Client/Frontend/Intro/IntroViewController.swift

```diff
@@ -33,10 +33,10 @@ struct IntroViewControllerUX {
     static let CardTitleCustomize = NSLocalizedString("Customize", tableName: "Intro", comment: "See http://mzl.la/1T8gxwo")
     static let CardTitleShare = NSLocalizedString("Share", tableName: "Intro", comment: "See http://mzl.la/1if9ODp")
     static let CardTitleChoose = NSLocalizedString("Choose", tableName: "Intro", comment: "See http://mzl.la/1if9ODp")
+    static let CardTitleSync = NSLocalizedString("Sync your devices", tableName: "Intro", comment: "See http://mzl.la/1if9ODp")
```

Comment:
We should have this string .. http://viewvc.svn.mozilla.org/vc/projects/l10n-misc/trunk/firefox-ios/en-US/firefox-ios.xliff?view=markup#l230

Unless it was removed.